client/stylesheets/_toCat.scss

```diff
@@ -3678,6 +3678,22 @@ On click the images will be displayed at normal size to complete the effect
     border: 2px solid #f8504b;
 }
 
+.profileHistoryCardUl {
+    background-color: gainsboro;
+}
+
+.profileHistoryTimestamp {
+    border: none !important;
+}
+
+.profileHistoryTimestamp::hover {
+      border: none !important;
+}
+
+.profileCardLi {
+     background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 5 spaces